internal/local/command_kafka_start.go

```diff
@@ -37,14 +37,14 @@ type ImagePullResponse struct {
 func (c *Command) newKafkaStartCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "start",
-		Short: "Start a single-node instance of Apache Kafka.",
+		Short: "Start a local Apache Kafka instance.",
 		Args:  cobra.NoArgs,
 		RunE:  c.kafkaStart,
 	}
 
-	cmd.Flags().String("kafka-rest-port", "8082", "The port number for Kafka REST.")
-	cmd.Flags().String("plaintext-port", "", "The port number for plaintext producer and consumer clients. If not specified, a random free port will be used.")
-
+	cmd.Flags().String("kafka-rest-port", "8082", "Kafka REST port number.")
+	cmd.Flags().StringSlice("plaintext-ports", nil, "A comma-separated list of port numbers for plaintext producer and consumer clients for brokers. If not specified, random free ports will be used.")
+	cmd.Flags().Int32("brokers", 1, "Number of brokers in the Confluent Local cluster.") // range: [1, 4]? do we enforce a range?
```

Comment:
:eyes:

Reply:
Haven't decided on the value we should set here. @davetroiano what's a proper range of number of brokers that users should be able to set?